### PR TITLE
Dynamic Landing Page

### PR DESCRIPTION
**Pull request by AlekseyGrimm**
https://github.com/rolling-scopes-school/tasks/blob/master/tasks/stage-0/projects.md#task-2-dynamic-landing-page-30

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+}
+
+// Set Background and Greeting
+function setBgGreet() {
+  let today = new Date(),
+    hour = today.getHours();
+
+  if (hour < 12) {
+    // Morning
+    document.body.style.backgroundImage = "url('https://img.huffingtonpost.com/asset/5e0f68ec2500003b1998fb2e.jpeg?cache=YqiWjN9UVt&ops=1778_1000')";
```
Comment (y-solovey) [inline script.js]: lets move images in the local folder repository

Comment (y-solovey) [reply script.js]: try to use CSS rules
background-size: cover; or
background-size: contain;

File: style.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    text-align: center;
+    height: 100vh;
+    color: black;
+  }
+  
+  #time {
+    font-size: 8rem;
+  }
+  
+  h1 {
```
Comment (y-solovey) [inline style.css]: a good practice is to not change default styles for attributes like H1, just add classes with styles

File: index.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Dynamic Page by Stalpetc</title>
+    <link rel="stylesheet" href="style.css"/>
+ </head>
+<body>
+    <time id="time">10:10:15</time>
+    <h1>
+      <span id="greeting">Good Aftermoon</span>
```
Comment (y-solovey) [inline index.html]: please fix placeholders

File: script.js (hunk 1 above)
Comment (AlekseyGrimm) [reply script.js]: well done

File: style.css (hunk 2 above)
Comment (AlekseyGrimm) [reply style.css]: done

File: index.html (hunk 3 above)
Comment (AlekseyGrimm) [reply index.html]: done

**Pull request closed without merging**